src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.1.0
+-----
+
+ * Using callable strings as `empty_data` in `ChoiceType` has been deprecated in Symfony 4.1 use a `\Closure` instead.
```

Comment:
The version information is redundant. To match the message style of other entries in this file, I'd suggest:

```
* deprecated support for callable strings as `empty_data` in `ChoiceType`
```